delivery.yaml

```diff
@@ -21,19 +22,26 @@ pipeline:
         IMAGE="${IMAGE_REGISTRY}/teapot/skipper:${RELEASE_VERSION}"
         ARM_IMAGE="${IMAGE_REGISTRY}/teapot/skipper-armv7:${RELEASE_VERSION}"
         ARM64_IMAGE="${IMAGE_REGISTRY}/teapot/skipper-arm64:${RELEASE_VERSION}"
+        MULTIARCH_IMAGE="${MULTIARCH_REGISTRY}/teapot/skipper:${RELEASE_VERSION}"
       else
         IMAGE="${IMAGE_REGISTRY}/teapot/skipper-test:${CDP_BUILD_VERSION}"
         ARM_IMAGE="${IMAGE_REGISTRY}/teapot/skipper-armv7-test:${CDP_BUILD_VERSION}"
         ARM64_IMAGE="${IMAGE_REGISTRY}/teapot/skipper-arm64-test:${CDP_BUILD_VERSION}"
+        MULTIARCH_IMAGE="${MULTIARCH_REGISTRY}/teapot/skipper-test:${CDP_BUILD_VERSION}"
       fi
-      export IMAGE ARM_IMAGE ARM64_IMAGE
+      export IMAGE ARM_IMAGE ARM64_IMAGE MULTIARCH_IMAGE
 
       make deps cicheck staticcheck gosec
       git status
       git diff
-      cd packaging && make docker.build.amd64 && git status && git diff && make docker.push.amd64
+
+      cd packaging
+      make docker.build.amd64 && git status && git diff && make docker.push.amd64
+      make docker.push.multiarch
```

Comment:
This builds and pushes a test image.

Reply:
The use of `&&` seems arbitrary - should we maybe `&&` all of the commands or have each on the single line for clarity?

Reply:
I think besides this here everything looks fine

Reply:
I separated the `cd packaging` from this particular command because _all_ subsequent commands expect to be run within that folder. That was hard to spot when it's lumped together with the first build.

I'm going with `&& \` all commands then because that's what you both agree on. But I wonder: putting each command on its own line should have the same affect since the pipeline will stop when one command fails.